app/assets/stylesheets/contactPages.scss

```
@@ -0,0 +1,36 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* universal */
+body {
+    padding-top: 60px;
+}
+section {
+    overflow: auto;
+}
```

```suggestion
body {
    padding-top: 60px;
}

section {
    overflow: auto;
}
```
Thêm line trống để clear code hơn e nha, tương tự những chỗ khác